testing/py3-pydocstyle/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname="py3-pydocstyle"
+_pkgname="pydocstyle"
+pkgver="4.0.1"
+pkgrel=0
+pkgdesc="Static analysis tool for checking compliance with Python docstring conventions"
+url="https://pydocstyle.org/"
+arch="noarch"
+license="MIT"
+depends="python3 py3-snowballstemmer"
+makedepends="python3 py3-pytest py3-mock py3-six"
+source="$_pkgname-$pkgver.tar.gz::https://github.com/PyCQA/$_pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	# integration tests call pip which doesn't exist
+    PYTHONPATH="$builddir/build/lib" py.test --deselect src/tests/test_integration.py -vv src/tests
```

Comment:
fwiw you've accidentally used spaces instead of tabs here.

Reply:
Oops. need to fix my editor to show this.